src/feedlistformaction.cpp

```diff
@@ -406,7 +406,9 @@ bool FeedListFormAction::process_operation(Operation op,
 	}
 	break;
 	case OP_MARKALLFEEDSREAD:
-		if (v->confirm(_("Do you really want to mark all feeds as read (y:Yes n:No)? "),
+		if (!get_cfg()->get_configvalue_as_bool(
```

Comment:
This fails to compile, can you please fix that? Here's the build log from the CI: https://travis-ci.com/github/newsboat/newsboat/jobs/394842896#L611